custom/icds_reports/models/aggregate.py

```diff
@@ -1587,6 +1589,32 @@ class Meta(object):
     _agg_atomic = False
 
 
+class AggregateAvailingServiceForms(models.Model, AggregateMixin):
```

Comment:
This table is not used anywhere so fine for this PR. but its generally a good practice to separate out the UCR and its usage table in two different Prs